package.json

```diff
@@ -1,27 +1,21 @@
 {
 	"name": "denon-receiver",
-	"version": "1.0.3",
-	"api_version": "1.0.0",
-	"keywords": [
-		"Protocol",
-		"Audio"
-	],
-	"manufacturer": "Denon",
-	"product": "AV Receiver",
-	"shortname": "denon",
-	"description": "Denon plugin for Companion",
-	"main": "index.js",
+	"version": "2.0.0",
+	"main": "main.js",
 	"scripts": {
-		"test": "echo \"Error: no test specified\" && exit 1"
+		"format": "prettier -w ."
 	},
-	"author": "Jeffrey Davidsz <jeffrey.davidsz@vicreo.eu>",
 	"license": "MIT",
-	"homepage": "https://github.com/bitfocus/companion-module-denon-receiver#readme",
-	"bugs": {
-		"url": "https://github.com/bitfocus/companion-module-denon-receiver/issues"
-	},
 	"repository": {
 		"type": "git",
 		"url": "git+https://github.com/bitfocus/companion-module-denon-receiver.git"
-	}
+	},
+	"dependencies": {
+		"18": "^0.0.0",
```

Comment:
What is this for?

Reply:
I haven't the slightest clue where that came from -- removed!